tests/unit/environments/BinaryLabelData.h

```diff
@@ -0,0 +1,86 @@
+#ifndef BINARY_LABEL_DATA_HPP
+#define BINARY_LABEL_DATA_HPP
+
+#ifdef HAVE_LAPACK
+
+#include <shogun/features/DataGenerator.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/labels/BinaryLabels.h>
+
+using namespace shogun;
+#ifdef USE_GPL_SHOGUN
+class BinaryLabelDatas
+{
+public:
+	BinaryLabelDatas(const int32_t num_samples)
+	{
+		SGMatrix<float64_t> data =
+		    CDataGenerator::generate_gaussians(num_samples, 2, 2);
```

Comment:
since you are using gaussians for the data, this should be reflected in the name of the class i reckon. so something like:
`GaussianCheckerboard(num_samples, num_dims, num_labels);`